elliott/elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -150,7 +151,23 @@ def get_flaws(flaw_bug_tracker: BugTracker, tracker_bugs: Iterable[Bug], brew_ap
 def _update_advisory(runtime, advisory, flaw_bugs, bug_tracker, noop):
     advisory_id = advisory.errata_id
     errata_config = runtime.get_errata_config()
-    cve_boilerplate = errata_config['boilerplates']['cve']
+
+    # Get the advisory name, eg. image|rpm|metadata|microshift
+    releases_config = runtime.get_releases_config()
+    advisories = releases_config[runtime.assembly]['assembly']['group']['advisories']
+    art_advisory_key = None
+    for _name, _id in advisories.items():
+        if _id == advisory_id:
+            art_advisory_key = _name
+            break
```

Comment:
```suggestion
    art_advisory_key = next((name for name, id in advisories.items() if id == advisory_id), None)
```

Reply:
i typed it out, do test before committing :)

Reply:
TIL!

updated in https://github.com/openshift-eng/art-tools/pull/1573/commits/544a92c5f8588db85ac8199fc945d3c181eb2b3c